src/framework/Cron.ts

```diff
@@ -62,6 +65,24 @@ export class Cron extends Base {
       await this.handler({ date, client: bot.client, logger });
     } catch (error) {
       logger.error(error, 'cron handler error');
+      try {
+        await findTextChannelByName(bot.client.channels, 'logs').send(
+          new MessageEmbed()
+            .setTitle('Cron run failed')
+            .setDescription(
+              dedent`
+              Name: ${this.name}
+              Run id: ${id}
```

Comment:
Pourquoi ne pas utiliser 2 inline fields ?

Reply:
je ne sais pas ce que ça veut dire

Reply:
`.setField('Name', this.name, true /* inline */).setField('Run id', id, true /* inline */`)
Ce qui donne un truc comme ça :
```
Cron run failed

Name          Run id
EpicGames     184ue-5r8de9j8e-t58frj8305-r8dwji

\```
...stack
\```
```